sdk/storage/Azure.Storage.Common/tests/Shared/TenantConfiguration.cs

```diff
@@ -123,7 +123,7 @@ public static TenantConfiguration Parse(string text)
             var values = text?.Split('\n');
             if (values == null || values.Length != 21)
             {
-                throw new ArgumentException();
+                throw Errors.InvalidArgumentParsing();
```

Comment:
Is this used anywhere other than our tests?  I don't think we want to add test specific code to the client library.

Reply:
Updated. Removed any errors made that are test specific.